### PR TITLE
Add amstream

### PR DESCRIPTION
**Pull request by Zemogiter**

### CODE REVIEW

**Comment by Zemogiter** [comment]
Ready for review.

**Comment by Zemogiter** [comment]
I would but POL5 fails to launch (both the .deb version and .zip one) after pulling the latest update from git.

**Comment by plata** [comment]
I think @madoar fixed this already. I will review his PR now.

File: Engines/Wine/Verbs/amstream/script.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    }
+    this.overrideDLL()
+        .set("native,builtin", ["amstream"])
+        .do();
+
+    return this;
+};
+
+/**
+ * Verb to install all the necessary things to run winevulkan (even inside wine mainline or newest wine-staging)
```
Comment (plata) [inline Engines/Wine/Verbs/amstream/script.js]: copy paste

**Pull request merged**